tests/integration/models/hostapp.spec.ts

```diff
@@ -74,5 +74,78 @@ describe('Hostapp model', function () {
 			]);
 			expect(firstRes).to.equal(secondRes);
 		});
+
+		it('should cache the results when providing extra options', async () => {
+			const firstRes = await balena.models.hostapp.getAllOsVersions(
+				['raspberrypi3'],
+				{ $filter: { is_invalidated: false } },
+			);
+			const secondRes = await balena.models.hostapp.getAllOsVersions(
+				['raspberrypi3'],
+				{ $filter: { is_invalidated: false } },
+			);
+			expect(firstRes).to.not.equal(secondRes);
+		});
```

Comment:
This test is a bit weird, this should cache the results but check for not equal. Is it strange only for me ?

Reply:
Yea you are right @JSReds :sweat: The description should be saying that it **shoul not**